gsa/src/web/utils/render.js

```diff
@@ -31,6 +31,8 @@ import {typeName, getEntityType} from 'gmp/utils/entitytype';
 export const UNSET_VALUE = '0';
 export const UNSET_LABEL = '--';
 
+export const NULL_VALUE = null;
```

Comment:
Hmm maybe we should just redefine UNSET_VALUE to null? This could possible break gsad based gmp but who cares ;-)